envi/__init__.py

```diff
@@ -44,39 +44,59 @@
 
 ARCH_MASK        = 0xffff0000   # Masked into IF_FOO and BR_FOO values
 
+# Code Maturity Ratings.  Also used for Unittest categories
```

Comment:
I'm rethinking the whole "Rating" scheme and I'm not a fan. It still feels too squishy,hand-wavey, and ultimately not particularly useful.  I feel like we should stick more with semver instead, which allows us to differentiate API levels/compliance and give a more fine grained notation of maturity than just "yup, it's mature".

Plus, there's more tooling that I can use like `bump2version` to increment architecture versions.

Also, we should probably call this `Architecture Maturity` instead.

Reply:
i'm ok with tracking these in some semver fashion, as long as each architecture and component gets its own.

do you mean starting them at some arbitrary number for each and incrementing them as we see fit?

care to suggest something?

Reply:
sorry, i'm forgetting some of the details from our last conversation about this....

so, i'm good with  v1.0.0 for amd64/i386, and i could be ok with v0.0.1 for PPC (although v0.9.0 might be more accurate with the level of effort that's been put into PPC over the past couple years).

i could explain what msp430 should be... but here's the thing.  i'm still pushing for versioning for the different components:
* Disasm
* Emu
* Symboliks
* Unittests
* (possibly) Analysis Modules

do we start the new architectures at v0.0.1 for each?
or do we make the numbers mean a little bit of our confidence level in them?